src/cljs/sixsq/slipstream/webui/main/views.cljs

```diff
@@ -84,7 +84,8 @@
          [ui/MenuItem {:fitted true}
           [authn-views/authn-menu]]]]
 
-       [messages/message-modal]])))
+       [messages/alert-slider]
+       [messages/alert-modal]])))
```

Comment:
what is alert-modal for?

Reply:
The slider just provides the title and status.  The details can be found in the modal by clicking the slider notification or the entry in the list on the messages page.

Reply:
BTW, the details are important when, for example, creating an API key/secret pair.